worlds/tunic/__init__.py

```diff
@@ -109,6 +109,12 @@ class TunicWorld(World):
     ut_can_gen_without_yaml = True  # class var that tells it to ignore the player yaml
 
     def generate_early(self) -> None:
+        try:
+            int(self.settings.disable_local_spoiler)
+        except AttributeError:
+            raise Exception("You have a TUNIC APWorld in your lib/worlds folder and custom_worlds folder.\n"
+                            "This would cause an error at the end of generation.\n"
```

Comment:
```suggestion
                            "This will cause an error at the end of generation.\n"
```

Reply:
It's saying "would" because it would do it if this exception wasn't raised -- the exception stops generation right then and there